data/human/ships.txt

```diff
@@ -2608,6 +2608,64 @@ ship "Nest"
 
 
 
+ship "Nighthawk"
+	sprite "ship/pirate nighthawk"
+	thumbnail "thumbnail/pirate nighthawk"
+	attributes
+		category "Transport"
+		"cost" 2530000
+		"shields" 4000
+		"hull" 1100
+		"required crew" 4
+		"bunks" 28
+		"mass" 267
+		"drag" 5.2
+		"heat dissipation" .65
+		"fuel capacity" 700
+		"cargo space" 16
+		"outfit space" 362
+		"weapon capacity" 102
+		"engine capacity" 123
+		weapon
+			"blast radius" 60
+			"shield damage" 600
+			"hull damage" 300
+			"hit force" 900
+	outfits
+		"Quad Blaster Turret" 2
+		"Heavy Anti-Missile Turret"
+		
+		"RT-I Radiothermal"
+		"KP-6 Photovoltaic Array" 3
+		"LP072a Battery Pack"
+		"D23-QP Shield Generator"
+		"Large Radar Jammer"
+		"Tactical Scanner"
+		"Laser Rifle" 24
+		"Fragmentation Grenades" 4
```

Comment:
```suggestion
		"Nerve Gas" 10
```
These are pirates, they should have pirate equipement?

Reply:
Pirates do sell Frag Grenades so they very much also use them but having player steal this ship and somehow get fined sounds fun.